postreview/templates/postreview/review_delete.html

```diff
@@ -0,0 +1,15 @@
+{% extends 'base.html' %}
+
+{% block content %}
+
+
+<form action='.' method='POST'>{% csrf_token %}
+    <h1>Do you want to delete the review "{{ object.first_name }}"?</h1>
+    <p><input type='submit' value='Yes' />  <a href='http://127.0.0.1:8000/'>Cancel</a></p>

```

Comment:
Try writing links using the django way i.e. in the {% url .. %} format, we don't use the localhost address in links.